platforms/linux/qlcplus.spec

```diff
@@ -41,6 +41,10 @@ of other lighting control commercial softwares.
 %build
 sed -ie '/UDEVRULESDIR/s|/etc/udev/rules.d|/usr/lib/udev/rules.d|' variables.pri
 
+# qmake-qt5 will only include existing files in install_translations - create the .qm files before *really* running qmake-qt5
+qmake-qt5
+make translate
```

Comment:
I think you can replace these 2 lines with
./translate.sh

Reply:
Right.
But even more, shouldn't it be able to generate the full Makefile directly after a clean checkout?

Reply:
I read the qtmake documentation and googled around. I now understand that there is no way to do this cleanly with qmake.

With qmake not being able to control the whole build process and quitly depending on build artifacts at configuration time, it becomes even more important to consistently do the steps in scripts and documentation. And to have an initial distclean that cleans everything up so qtmake will do exactly the same every time.